Dronecode-SDK-Swift/Calibration.swift

```diff
@@ -249,21 +249,21 @@ public class Calibration {
     private func createCalibrateMagnetometerObservable() -> Observable<ProgressData> {
         return Observable.create { observer in
             let calibrateMagnetometerRequest = DronecodeSdk_Rpc_Calibration_SubscribeCalibrateMagnetometerRequest()
-
+            print("createCalibrateMagnetometerObservable start")
```

Comment:
You probably wanted to remove that line again?

Reply:
Oh, I missed that one. Thank you.